3회차/윤효근/5_신용카드만들기2.py

```diff
@@ -1,3 +1,13 @@
 import sys
 
-sys.stdin = open("_신용카드만들기2.txt")
+sys.stdin = open("신용카드 만들기 2.txt", "r")
+
+T = int(input())
+
+for test_case in range(1, T + 1):
+    n= list(map(str,input()))
+
+    if (n[0] == '3' or n[0] == '4' or n[0] == '5' or n[0] == '6' or n[0] == '9') and len(n) - n.count('-') == 16: #3 4 5 6 9로 시작하지 않으면 0출력
```

Comment:
좋은 풀이네요 😄 
`if` 조건문에 `in` 을 사용한 풀이도 가능합니다!
```suggestion
    if (n[0] in '34569') and len(n) - n.count('-') == 16: #3 4 5 6 9로 시작하지 않으면 0출력
```